src/PrestaShopBundle/Entity/Repository/FeatureFlagRepository.php

```diff
@@ -53,6 +55,25 @@ public function getByName(string $featureFlagName): ?FeatureFlag
         return $this->findOneBy(['name' => $featureFlagName]);
     }
 
+    /**
+     * Get handler to use for this FeatureFlag
+     *
+     * @param string $featureFlagName
+     *
+     * @return \PrestaShop\PrestaShop\Core\FeatureFlag\TypeHandlerInterface
+     *
+     * @throws \PrestaShopException
+     */
+    public function getHandler(string $featureFlagName): TypeHandlerInterface
+    {
+        $featureFlag = $this->getByName($featureFlagName);
+        if (null === $featureFlag) {
+            throw new \InvalidArgumentException('FeatureFlag ' . $featureFlagName . ' not found.');
+        }
+
+        return HandlerFactory::getHandler($featureFlag);
+    }
```

Comment:
Why this method. IMO we should retrieve a Handler directly by the HandlerFactory.